src/components/pie-author/pie-author.tsx

```diff
@@ -388,6 +398,20 @@ export class Author {
     return newConfig;
   }
 
+  removeRubricItemTypes(pieContentModel, rubricElements) {
+    if (!rubricElements.length || !pieContentModel.models) {
+      return pieContentModel;
+    }
+
+    // delete the rubric elements from elements object
+    rubricElements.forEach(rubricElementKey => delete pieContentModel.elements[rubricElementKey]);
```

Comment:
rubricElements is an array with rubric ids. 'elements' don't have access to ids, but the keys